tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5792,6 +5792,44 @@ def _impl(ctx):
         bitcode_embedded_markers_feature = feature(name = "bitcode_embedded_markers")
         bitcode_embedded_feature = feature(name = "bitcode_embedded")
 
+    if (ctx.attr.cpu == "armeabi-v7a" or
+        ctx.attr.cpu == "darwin_x86_64" or
+        ctx.attr.cpu == "ios_arm64" or
+        ctx.attr.cpu == "ios_arm64e" or
+        ctx.attr.cpu == "ios_armv7" or
+        ctx.attr.cpu == "ios_i386" or
+        ctx.attr.cpu == "ios_x86_64" or
+        ctx.attr.cpu == "tvos_arm64" or
+        ctx.attr.cpu == "tvos_x86_64" or
+        ctx.attr.cpu == "watchos_arm64_32" or
+        ctx.attr.cpu == "watchos_armv7k" or
+        ctx.attr.cpu == "watchos_i386" or
+        ctx.attr.cpu == "watchos_x86_64"):
+        generate_linkmap_feature = feature(
+            name = "generate_linkmap",
+            flag_sets = [
+                flag_set(
+                    actions = [
+                        ACTION_NAMES.objc_executable,
+                        ACTION_NAMES.objcpp_executable,
+                    ],
+                    flag_groups = [
+                        flag_group(
+                            flags = [
+                                "-Xlinker",
+                                "-map",
+                                "-Xlinker",
+                                "LINKMAP_EXEC_PATH=%{linkmap_exec_path}",
```

Comment:
Can you pass this path directly instead of having the wrapper expand it?

Reply:
This path comes directly from bazel as a build var-- that is, it will already be expanded once it hits the wrapper.

I think you need bazel to tell you the path as it inserts the artifact into the correct output group?

Reply:
Ah right. Why am I adding a no-op to the wrapper...

Updated. PTAL.